x-pack/plugins/metrics_entities/README.md

```diff
@@ -0,0 +1,34 @@
+# metrics_entities
+
+Metrics entities where you can add transforms for metrics and entities.
+
+The transform jobs are on the file system as:
+ - modules (The different logical pieces)
+  
+When you create a "group" of transforms you can create them using an optional prefix to separate things based on queries, security concerns, or different Kibana spaces.
```

Comment:
Will want to expand upon setup instructions and detail what feature flags need to be flipped to enable. Also will want to add a section for _disabling_ and detailing any manually steps necessary to perform cleanup of any lingering transforms (since those aren't cleaned up automatically on disable).

Reply:
Totally agree, adding it to my current TODO list. Might not happen in this PR right now but should happen as I flush out those details.